docs/installation.rst

```diff
@@ -77,3 +77,5 @@ If you don't want to (or cannot) use Virtualenv, the best idea is to install Cek
 .. include:: dependencies.rst
 
 .. include:: upgrade.rst
+
+.. include:: CONTRIBUTING.rst
```

Comment:
I don't think this should be part of the installation document.

Reply:
Perhaps move it to the top level then?

Reply:
Yes, this should be better.